Mono.Debugging/Mono.Debugging.Client/StackFrame.cs

```diff
@@ -20,7 +20,7 @@ public class StackFrame
 		bool isExternalCode;
 		bool isDebuggerHidden;
 		bool hasDebugInfo;
-		string fullModuleName;
+		protected string fullModuleName;
```

Comment:
This should use proper naming style conventions.

```suggestion
		protected string FullModuleName;
```

Reply:
Better use a protected property instead of a field.

Reply:
I wanted to minimize changes, also FullModuleName is already a property, so it would mean making a backing field of a property itself a property. We can't use upper case because it would conflict with the FullModuleName property.

Where in the naming conventions do we have specified that protected fields need to start with an uppercase?

Reply:
Decided to go with an auto-property.